app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -297,7 +297,6 @@ class RawSubmissionTable extends React.Component {
 
 let SubmissionTable = withSelection(RawSubmissionTable);
 SubmissionTable.defaultProps = {
-  is_admin: false,
```

Comment:
Since this component uses defaultProps, we should specify a new default value for the prop you added. (Set it to `false`)